docs/api-testing/mark3/mocking.md

```diff
@@ -0,0 +1,271 @@
+Run it with the `--validate-request` switch to activate the validation of inbound requests.
+
+Whenever a request is performed, it will be validated against the schema, and if a mismatch is present, an error like the following will be returned:
+
+```json
+{
+    "collection": "queryParams",
+    "errors": [
+        {
+            "argument": [
+                "integer"
+            ],
+            "instance": "aa",
+            "message": "is not of a type(s) integer",
+            "name": "type",
+            "path": [],
+            "property": "instance",
+            "schema": {
+                "type": "integer"
+            },
+            "stack": "instance is not of a type(s) integer"
+        }
+    ],
+    "message": "Wrong field types"
+}
+```
+
+The response will also contain the x-sauce-error: true header, signifying that the response is not mocked, but it's an internal error.
+
+
+### Dynamic examples
+The system allows for examples containing dynamic data using the Handlebars markup. Remember that if you use dynamic examples in your OpenAPI specs, your spec will reduce its usability for documentation purposes as documentation renderers don't support it.
+
+To have dynamic parameters, you simply place an expression between double curly brackets as in `{{requestUrl}}`.
+
+The available objects in the scope are the same as the ones used by x-sauce-cond, so: `uriParams`, `queryParams`, `headers`, `body`.
+
+As an example, the following template will echo the shape of the request back in the response:
+
+```json
+{
+ "url":"{{requestUrl}}",
+ "requestHeaders": {{json headers}},
+ "requestBody": {{json body}},
+ "ipAddress": "{{ipAddress}}"
+}
+```
+
+The "json" keyword will convert a full data structure into its JSON equivalent.
+
+
+
+
+## E2E Mode
+When Piestry is run with `--e2e`, it will turn into a reverse proxy gateway and forward the requests based to the origin, according to the OpenAPI specification. The requirement is the "server" definition of the OpenAPI spec should lead to an actual location.
+
+In this mode, you can enable contract validators as well as capture mode.
+
+### Contract Validators
+There are two types of validations you can activate, focusing on different areas.
+
+### Validate Response
+The validate response is similar to the ["validate examples" (mocking mode)](/api-testing/mark3/mocking/#validate-examples); the difference is that will validate the actual responses in an end-to-end session. Use the switch `--validate-response` to enable it.
+
+### Capture Mode
+Capture mode is activated by passing the `--capture` parameter, followed by the path to a directory. As the requests go through, Piestry will capture the responses coming from the origin and save them to file.
+
+When `--capture` is executed without `--e2e`, Pietry will try to map the saved files to the OpenAPI definition and serve them as examples.
+
+
+
+## Logger
```

Comment:
This section is ok as "hint" for the user but we need a separate page for the Logger

Reply:
done!